src/main/resources/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder/tasks-top-extra.jelly

```diff
@@ -30,10 +30,10 @@ THE SOFTWARE.
     <j:if test="${computation != null}">
         <j:set var="title" value="${computation.displayName}"/>
     </j:if>
-    <l:task icon="images/24x24/notepad.png" href="${url}/computation/" title="${title}">
+    <l:task icon="images/24x24/notepad.png" href="${computation.getUrl()}" title="${title}">
```

Comment:
:bug: if you are using `computation` then you must enclose all this in the `if` block that checks that `computation != null` (which also means that the `title` local variable is superfluous: you can inline `${computation.displayName}`).

Probably the previous code was producing a link which would be a 404 if clicked. Not sure if you would ever really run into this—from reading the code, it seems `getComputation()` is nonnull after the constructor has completed.